cobigen-plugins/cobigen-javaplugin-parent/cobigen-javaplugin/src/main/java/com/devonfw/cobigen/javaplugin/merger/JavaMerger.java

```diff
@@ -71,6 +73,10 @@ public String getType() {
   @Override
   public String merge(File base, String patch, String targetCharset) throws MergeException {
```

Comment:
We should create a new method for that I think. Otherwise we would have to adjust the javadoc description and explain, what happens if path is null.